src/main/target/SPRACINGF3NEO/config.c

```diff
@@ -68,8 +68,6 @@ static targetSerialPortFunction_t targetSerialPortFunction[] = {
 
 void targetConfiguration(void)
 {
-    barometerConfigMutable()->baro_hardware = BARO_DEFAULT;
```

Comment:
You can probably remove the includes for barometer and compass above as well.

Reply:
@mikeller done, removed some other ones too. LGTM!